package.json

```diff
@@ -29,12 +29,24 @@
   "scripts": {
     "build": "webpack --config webpack.config.js",
     "build:dev": "yarn build --mode=development",
-    "build:css": "postcss ./app/assets/stylesheets/application.postcss.css -o ./app/assets/builds/application.css"
+    "build:css": "postcss ./app/assets/stylesheets/application.postcss.css -o ./app/assets/builds/application.css",
+    "lint:js": "prettier --check app/javascript",
+    "lint:css": "prettier --check app/assets/stylesheets",
+    "prettify": "prettier --write app/javascript app/assets/stylesheets",
+    "prepare": "husky install"
   },
   "devDependencies": {
-    "pa11y-ci": "^3.0.1"
+    "husky": ">=6",
+    "lint-staged": ">=10",
+    "pa11y-ci": "^3.0.1",
+    "prettier": "2.6.1"
   },
   "resolutions": {
     "netlify-cms-app/**/trim": "0.0.3"
+  },
+  "lint-staged": {
+    "*.{jsx?,s?css}": "prettier --write",
```

Comment:
[issue] something about this isn't writing properly. I changed a JS file to have single quotes and then the following things happened:

1. `git commit -am "test"` showed the hooks running, but allowed the updated file to be committed (bad)
2. `yarn lint:js` returned a failure (good)
3. `yarn prettify` switched the file back to double quotes (good)

I would expect that the hook would either do the behavior of step 2 or 3 there rather than letting the bad code through.